examples/k3s/main.tf

```diff
@@ -157,7 +157,7 @@ resource "aws_security_group" "security_group" {
     description = "Allow internal traffic"
     from_port   = 0
     to_port     = 0
-    protocol    = "tcp"
+    protocol    = "-1"
```

Comment:
Would you mind updating the basic config to use this as well? I was thinking we should test for specific protocol (e.g., tcp + udp) but my second thought was that it's the internal network - we should allow all things. Do you agree?

Reply:
Yes, I can do that. Regarding the testing, I've done some tests - enabling both the tcp and udp. I faced same issue.

Reply:
okay - then let's just keep at -1, I do think it's OK to enable all protocols for the internal traffic!